.github/workflows/tests.yml

```diff
@@ -7,23 +7,24 @@ on:
 jobs:
   rake:
     runs-on: "ubuntu-latest"
-    continue-on-error: true
     strategy:
       fail-fast: false
       matrix:
         ruby_version:
-          - "2.3.8"
-          - "2.4.5"
-          - "2.5.3"
-          - "2.6.3"
-          - "2.7.2"
+          - "2.3"
+          - "2.4"
+          - "2.5"
+          - "2.6"
+          - "2.7"
           - "3.0"
+          - "3.1"
+          - "3.2"
           - "jruby-9.2.19.0"
-          - "truffleruby-21.1.0"
+          - "truffleruby-22.3.0"
     steps:
-      - uses: "actions/checkout@v2"
+      - uses: "actions/checkout@v3"
       - uses: "ruby/setup-ruby@v1"
         with:
           ruby-version: "${{ matrix.ruby_version }}"
-      - run: "bundle install"
+          bundler-cache: true
```

Comment:
Allow the Setup Ruby action to handle the `bundle install`. This improves the performance as it can cache gems between builds.

Reply:
👍